### DISCUSSION TITLE
DIANN output files - "diann-output.tsv" vs  "diann-output.pg_matrix.tsv"

### DISCUSSION THREAD

**Discussion by MRamirezGarras** [vdemichev/DiaNN#665] [Q&A] [answered]
I am trying to establish a pipeline to work with DIA-NN and Perseus. I have done it by extracting the MaxLFQ values from the file "diann-output.tsv" using DIA-NN R package and formatting in a way that it can be loaded into Perseus. 

I worked well, but then I noticed that the values in the file "diann-output.pg_matrix.tsv" are very similar to the MaxLFQ values I got. The striking thing is that from some of the proteins the values were exactly the same in both files, while for other proteins there was a slight variation. Besides, when I extracted the values from diann-output.tsv I got around 10% more proteins. 

I would rather work with the values from "diann-output.pg_matrix.tsv" and skip the R step, but I am not sure this is the right way. Anybody knows the difference between the values in these two files is?

**Comment by vdemichev** [comment]
diann-output.pg_matrix.tsv should match specifically the PG.MaxLFQ column. The main report is not filtered, unlike the pg_matrix, hence more proteins. I would recommend to always use the main report, this gives full control of the filtering.

Best,
Vadim

**Reply by vdemichev** [reply, marked as answer]
But yes, matrix can always also be used, if this is more convenient.
